src/components/annotate/sidePanel/Annotation.tsx

```diff
@@ -0,0 +1,200 @@
+/* Import Dependencies */
+import { useEffect, useRef } from 'react';
+import classNames from 'classnames';
+import Moment from 'moment';
+import KeycloakService from 'keycloak/Keycloak';
+import { Row, Col } from 'react-bootstrap';
+
+/* Import Store */
+import { useAppSelector, useAppDispatch } from 'app/hooks';
+import {
+    getAnnotateTarget, setEditAnnotation,
+    getHighlightAnnotationId, setHighlightAnnotationId
+} from 'redux/annotate/AnnotateSlice';
+
+/* Import Types */
+import { Annotation as AnnotationType } from 'global/Types';
+
+/* Import Sources */
+import AnnotationMotivations from 'sources/annotationMotivations.json';
+
+/* Import Styles */
+import styles from 'components/annotate/annotate.module.scss';
+
+/* Import Icons */
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faShareNodes, faPen, faTrashCan } from '@fortawesome/free-solid-svg-icons';
+import { faThumbsUp, faThumbsDown, faMessage } from '@fortawesome/free-regular-svg-icons';
+
+/* Import API */
+import DeleteAnnotation from 'api/annotate/DeleteAnnotation';
+
+
+/* Props Typing */
+interface Props {
+    annotation: AnnotationType,
+    UpdateAnnotationView: Function
+};
+
+
+const Annotation = (props: Props) => {
+    const { annotation, UpdateAnnotationView } = props;
+
+    /* Hooks */
+    const dispatch = useAppDispatch();
+    const annotationRef = useRef<HTMLDivElement>(null);
+
+    /* Base variables */
+    const annotateTarget = useAppSelector(getAnnotateTarget);
+    const highlightAnnotationId = useAppSelector(getHighlightAnnotationId);
+    const annotationMotivations = { ...AnnotationMotivations };
+    let annotationValue: string = '';
+
+    /* Check if Annotation value is multi value */
+    if (Array.isArray(annotation.body.value)) {
+        annotationValue = annotation.body.value.join(', ');
+    } else {
+        annotationValue = annotation.body.value;
+    }
+
+    /* Function for highlighting an Annotation if created or modified */
+    const HighlightAnnotation = () => {
+        const annotationDiv: HTMLDivElement = annotationRef.current as HTMLDivElement;
+
+        /* Reset hightlight annotation id */
+        dispatch(setHighlightAnnotationId(''));
+
+        /* Highlight 1 */
+        annotationDiv.classList.add(`${styles.highlight}`);
+
+        setTimeout(() => {
+            annotationDiv.classList.remove(`${styles.highlight}`);
+
+            setTimeout(() => { Second() }, 500);
```

Comment:
Is the second timeout needed?

Reply:
No, was already thinking about this. Replaced with an interval and timeout, which also loops the highlight three times in total.

Reply:
:+1: